### ISSUE TITLE
Group Registration Block No Longer Passes Querystring parameters to Workflows

### ISSUE THREAD

**Issue by JimMichael** [SparkDevNetwork/Rock#3786]
### Description
We have been using GroupRegistration.ascx block as a clever way to add a person _and_ their spouse to Rock (difficult if not impossible to do with a worfkflow user form), and launch a Workflow to do "other stuff" as part of a "connect card" process. We also pass a selected Campus value via the querystring parameter to the workflow using syntax like so: `https://myserver.org/PageWithGroupRegisterBlock?Campus=1` and as long as there's an _attribute_ in the workflow with a key that matches the parameter (in this case Campus), that info gets passed to the workflow.

This functionality broke in either 8.7 or 8.8 when modifications were made to this block. It now no longer allows you to pass attribute values to a workflow via the querystring. The attribute values with matching key in the WF simply remain empty when the WF is launched from the block. Trying to determine if this is an over-site or an intentional change in behavior.

### Steps to Reproduce
1. Create a simple Workflow with a single test attribute with a key of `SomeKey` that has a single _Complete Workflow_ action.
1. Add Group Register block to a page and configure it to point at any group, and to point at your new workflow.
1. Access the page via https://yourserver.org/page/123?SomeKey=TEST (where page 123 is the page with your Group Register block)
1. Submit the form and examine the new workflow instance. Observe that the SomeKey attribute is empty instead of containing TEST, which was passed from the Group Register querystring.

**Expected behavior:**

Querystring parameters could be passed in to workflow attributes.

**Actual behavior:**

This worked fine in 8.6 and prior, but no longer works from 8.8 through prealpha.

### Versions
* **Rock Version:** 8.8+

**Comment by JimMichael** [comment]
I've since found out that we had forked the original core block to make some cosmetic changes to it, and that's what we were using until recently when we tried to switch back to using the updated core block we now saw in 8.8, and discovered the querystring parameters no longer work. 

So while the issue is still true, we don't really know if it happened at 8.7/8.8 because we forked the core block long before that... probably close to a year before. So it could have been in 7.anything that the breaking change occurred.

**Comment by nairdo** [comment]
@JimMichael  Can you send me a link to a zip/code of your forked version?  (I'm looking back at very old versions from four years ago and I don't see how that would work.)

**Comment by JimMichael** [comment]
Let's pause this for a minute while we try to figure out what's going on on our end. What I *thought* was true when I entered this issue is not... here are the facts: We have a (mildly) customized GroupRegister block that somehow has the magical ability to pass querystring parameters to the launched Workflow and those parameters get stuffed into WF attributes with matching keys. The current core block, and one from a year ago and one from *2015* ALL do not have this behavior (whatever we pass on the querystring with those blocks does not make it into the workflow).

The logical conclusion is that we customized our own block to add this functionality... but the problem is that we have no such skills or memory of adding any features to this block. Our "customizations" to it involved simple things like hiding the Spouse fields on "Full with Spouse" mode behind a checkbox.... all front-end stuff. Do you still want me to send you our current block in case you can recognize how/why it has this "magic" querystring ability?

**Comment by nairdo** [comment]
> Do you still want me to send you our current block in case you can recognize how/why it has this "magic" querystring ability?

Yes, I think that would be quite helpful. :)

**Comment by JimMichael** [comment]
Ok, Nick took a look at our block and it is indeed far more custom than we realized... it has code that loops through page parameters and stuffs them into WF attributes that match. We still have no idea where we got this block (we don't have the chops to have done it ourselves) but we've had it so long that we assumed it was just a simple fork of the core block with our visual mods.

All that said, this is a false alarm and we'll have to see about getting the same changes into the newer core block. Sorry for the waste of time!
